<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background:rgb(255, 255, 255);
font-size:15px;
font-style:MingLiU-ExtB;
}

QLineEdit{
color:#8d98a1;
background-color:rgba(64, 83, 97, 230);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QLineEdit:hover{
color:#8d98a1;
background-color:rgba(64, 83, 97, 80);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QLineEdit:focus{
color:#8d98a1;
background-color:rgba(64, 83, 97, 180);
font-size:16px;
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}

QPushButton{
background:rgb(65, 175, 235);
color:rgb(0, 79, 148);
border-style:outset;
border-radius:10px;
font-style:MingLiU-ExtB;
}
QPushButton:hover{
background-color:rgba(80, 190, 250,130);
border-style:inset;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
background-color:rgba(80, 190, 250,255);
border-style:inset;
font-style:MingLiU-ExtB;
}

QPushButton#hideButton{
background: rgba(255, 255, 255, 0);
color: rgb(145, 145, 145);
font-style:MingLiU-ExtB;
}
QPushButton#hideButton:hover{
background: rgba(212, 212, 212, 100);
color: rgb(145, 145, 145);
font-style:MingLiU-ExtB;
}
QPushButton#hideButton:pressed{
background: rgba(212, 212, 212, 255);
color: rgb(145, 145, 145);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton{
background: rgba(255, 255, 255, 0);
color: rgb(218, 0, 0);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton:hover{
background: rgba(255, 0, 0, 100);
color: rgb(218, 0, 0);
font-style:MingLiU-ExtB;
}
QPushButton#closeButton:pressed{
background: rgba(255, 0, 0, 255);
color: rgb(218, 0, 0);
font-style:MingLiU-ExtB;
}
QCheckBox{
background:rgba(85,170,255,0);
color:white;
font-style:MingLiU-ExtB;
}

</string>
  </property>
  <widget class="QLabel" name="indowLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 250, 255);</string>
   </property>
   <property name="text">
    <string>  WeNoChat</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="chatName">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>841</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(235, 250, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>841</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,50);</string>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>0</y>
     <width>16</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255,50);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>841</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>841</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border:sold 10px rgba(0,0,0);
	background-color: rgb(245, 250, 255);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>821</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string>输入消息......</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>90</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QPushButton" name="biaoqingButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/biaoqing.png</normaloff>:/assets/biaoqing.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/jiandao.png</normaloff>:/assets/jiandao.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_send_image">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/zhaopian.png</normaloff>:/assets/zhaopian.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>35</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/wenjian.png</normaloff>:/assets/wenjian.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#listWidget{border-image: url(:/assets/preview.jpg);}</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="biaoqingFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>244</y>
      <width>311</width>
      <height>300</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(227, 248, 252, 235);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="bButton1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/微笑.jpg</normaloff>:/assets/emoji/微笑.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/好！.jpg</normaloff>:/assets/emoji/好！.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>0</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/大哭.jpeg</normaloff>:/assets/emoji/大哭.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton5">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/装傻.jpeg</normaloff>:/assets/emoji/装傻.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton6">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/点赞.jpeg</normaloff>:/assets/emoji/点赞.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/冷笑.jpeg</normaloff>:/assets/emoji/冷笑.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton8">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/求求你了.jpeg</normaloff>:/assets/emoji/求求你了.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/aaa.png</normaloff>:/assets/emoji/aaa.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>90</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../uires.qrc">
       <normaloff>:/assets/emoji/渴望.jpeg</normaloff>:/assets/emoji/渴望.jpeg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>90</width>
       <height>90</height>
      </size>
     </property>
    </widget>
    <zorder>bButton2</zorder>
    <zorder>bButton3</zorder>
    <zorder>bButton5</zorder>
    <zorder>bButton6</zorder>
    <zorder>bButton4</zorder>
    <zorder>bButton8</zorder>
    <zorder>bButton9</zorder>
    <zorder>bButton7</zorder>
    <zorder>bButton1</zorder>
   </widget>
   <widget class="QLabel" name="cover_label">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>591</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/assets/textwenochat.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>frame_2</zorder>
   <zorder>listWidget</zorder>
   <zorder>biaoqingFrame</zorder>
   <zorder>cover_label</zorder>
  </widget>
  <widget class="QPushButton" name="pushButton_addfriend">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="whatsThis">
    <string>加好友</string>
   </property>
   <property name="text">
    <string>添加好友</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="friendList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>211</width>
     <height>571</height>
    </rect>
   </property>
   <property name="headerHidden">
    <bool>true</bool>
   </property>
   <property name="expandsOnDoubleClick">
    <bool>false</bool>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="hideButton">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>—</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#pushButton_image{
background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../uires.qrc">
     <normaloff>:/assets/head.png</normaloff>:/assets/head.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>81</width>
     <height>81</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="setting">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>0</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#setting{
background-color: rgba(255, 255, 255, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../uires.qrc">
     <normaloff>:/assets/settings.jpg</normaloff>:/assets/settings.jpg</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addfriend_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="text">
    <string>创建群聊</string>
   </property>
  </widget>
  <zorder>chatName</zorder>
  <zorder>indowLabel</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>pushButton_addfriend</zorder>
  <zorder>friendList</zorder>
  <zorder>hideButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>pushButton_image</zorder>
  <zorder>frame</zorder>
  <zorder>setting</zorder>
  <zorder>pushButton_addfriend_2</zorder>
 </widget>
 <resources>
  <include location="../uires.qrc"/>
 </resources>
 <connections/>
</ui>
